diff --git a/.eslintrc.yml b/.eslintrc.yml
index 60ea3aa..800007b 100644
--- a/.eslintrc.yml
+++ b/.eslintrc.yml
@@ -19,6 +19,10 @@ plugins:
 rules:
   prettier/prettier:
     - error
+  "@typescript-eslint/no-unused-vars":
+  - warn
+  - argsIgnorePattern: "^_"
+
 settings:
   react:
     version: detect
diff --git a/src/openshift/pages/ExamplePage.tsx b/src/openshift/pages/ExamplePage.tsx
index d64f5f9..d8fe883 100644
--- a/src/openshift/pages/ExamplePage.tsx
+++ b/src/openshift/pages/ExamplePage.tsx
@@ -185,7 +185,8 @@ export default function ExamplePage() {
         <PageSection variant="light">
           {instance && instance.metadata ? (
             <Text>
-              Selected Cryostat "{instance.metadata.name}" in project "{instance.metadata.namespace}"
+              Selected Cryostat `&quot;`{instance.metadata.name}`&quot;` in project `&quot;`
+              {instance.metadata.namespace}`&quot;`
             </Text>
           ) : undefined}
           <Text>API Request Method</Text>
diff --git a/src/openshift/services/ApiService.tsx b/src/openshift/services/ApiService.tsx
index 28e97dc..0c774cd 100644
--- a/src/openshift/services/ApiService.tsx
+++ b/src/openshift/services/ApiService.tsx
@@ -34,7 +34,7 @@ export class ApiService {
     );
   }
 
-  cryostat(ns: string, name: string, method: string, requestPath: string, body?: object): Observable<string> {
+  cryostat(ns: string, name: string, method: string, requestPath: string, _body?: object): Observable<string> {
     const url = this.proxyUrl(`upstream/${requestPath}`);
     return from(
       consoleFetch(url.toString(), {
